index.js

```diff
@@ -126,26 +127,59 @@ function validate(word) {
   return true;
 }
 
-function stonksify(word) {
+function stonksify(word, showAllWords = true) {
   word = word.toLowerCase();
+  let possibleWords = [];
   console.groupCollapsed("Regex Details: " + word);
   for (let i = 0; i < stonksifyRules.length; i++) {
     let toReplace = new RegExp(stonksifyRules[i][0]);
     let replaceWith = stonksifyRules[i][1];
     let newWord = word.replace(toReplace, replaceWith);
+    if (possibleWords.indexOf(newWord) === -1) {
```

Comment:
index of is O(n) runtime, but if you use a set checking if the value is in there is O(1) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set